### ISSUE TITLE
Why is the position of the picker window weird? [Bug] 

### ISSUE THREAD

**Issue by s6677** [xmichelo/Beeftext#533]
Every time the shortcut key calls out the picker window, its default position is in the lower right corner. Is it deliberately set like this? It seems strange. If there are a lot of hot phrases, I can hardly see the content of the block, and it is easy to move every time. to the middle position

15.0portable version

**Comment by xmichelo** [comment]
Hi.

Picker Window should b positioned centered over the active window. What you are describing is not normal. Do you use any 3rd party software to manage your windows layout?

**Comment by s6677** [comment]
I'm not using a window program,， I tried it with older versions（for e.g 7.2
suppprt rich text）, they are displayed in the center, only the recent
latest versions are in the bottom right corner

Xavier Michelon ***@***.***> 于 2022年11月22日周二 下午2:52写道：

**Comment by xmichelo** [comment]
OK. That's strange. What Windows version are you using?

**Comment by s6677** [comment]
In my surface pro 2in1，display setting :text size, and application item size is 200%(in surface pro ,This is the best display,I can't change this setting and I don't know if changing this will make it work now) ，and resolution 2736*1824I  think that It has nothing to do with the Windows versionI think the  reason is text and item "200%",this looks a little special，，So display pickwindow is wrong ? maybe  you could  try to think this .发送自我的盖乐世
-------- 原始信息 --------发件人： Xavier Michelon ***@***.***> 日期: 2022/12/29  下午3:16  (GMT+09:00) 收件人： xmichelo/Beeftext ***@***.***> 抄送： s6677 ***@***.***>, Author ***@***.***> 主题： Re: [xmichelo/Beeftext] Why is the position of the picker window weird? [Bug]  (Issue #533) 
OK. That's strange. What Windows version are you using?

**Comment by burukan** [comment]
Window position calculation for scaled display seems to be wrong. Till its fixed one can fix this via the method I wrote on the other chat.

Start menu > beeftext > right click > open file location > beeftext > right click > properties > compatibility tab > "change high dpi settings" > "override high dpi scalin behavior" select "system" or "system (enchanced)" > ok > ok

quit beeftext from system tray and re-open the app... All should be back to normal.

**Comment by burukan** [comment]
@xmichelo I believe you can change windows scale anything higher than 100% to reproduce the issue 👍 (which is a must for 4k on small monitor these days)...

**Comment by xmichelo** [comment]
@burukan You're right, there is a problem with the computation of the picker window position when display is scaled. Thank you for pointing that out.
